erpnext_com/www/docs/user/manual/en/index.html

```diff
@@ -13,6 +13,17 @@
 		</nav>
 		<h1>User Manual (English)</h1>
 		<p class='lead mx-0'>Explore modules to delve deep into ERPNext features</p>
+		<form action="/search">
```

Comment:
@netchampfaris, kindly add the searchbar in all md pages of docs, will give better accessibility